### PR TITLE
Add set oracle signing address script

### PR DESCRIPTION
**Pull request by daveroga**
Script to change oracle signing address in CrossChainProofValidator

Example
```
npx hardhat run scripts/setOracleSigningAddress.ts --network zkevm-cardona
```

### CODE REVIEW

File: scripts/setOracleSigningAddress.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { ethers } from "hardhat";
+import { getConfig } from "../helpers/helperUtils";
+import { CONTRACT_NAMES } from "../helpers/constants";
+
+async function main() {
+  const oracleSigningAddress = "0xf0Ae6D287aF14f180E1FAfe3D2CB62537D7b1A82"; // production signing address
+
```
Comment (AndriianChestnykh) [inline scripts/setOracleSigningAddress.ts:7]: We still have the same address in constants, so better to export it as ORACLE_SIGNING_ADDRESS_PRODUCTION from there.

File: scripts/setOracleSigningAddress.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  const crossChainProofValidatorAddress = await state.getCrossChainProofValidator();
+  console.log(`CrossChainProofValidator address: ${crossChainProofValidatorAddress}`);
+
+  const crossChainProofValidator = await ethers.getContractAt(
+    CONTRACT_NAMES.CROSS_CHAIN_PROOF_VALIDATOR,
+    crossChainProofValidatorAddress,
+  );
+  const tx = await crossChainProofValidator.setOracleSigningAddress(oracleSigningAddress);
+
+  console.log(`Oracle signing address set to ${oracleSigningAddress} in tx ${tx.hash}`);
```
Comment (AndriianChestnykh) [inline scripts/setOracleSigningAddress.ts:23]: We may console log CrossChainProofValidator address as well

File: scripts/setOracleSigningAddress.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { ethers } from "hardhat";
+import { getConfig } from "../helpers/helperUtils";
+import { CONTRACT_NAMES, ORACLE_SA_PROD } from "../helpers/constants";
```
Comment (AndriianChestnykh) [inline scripts/setOracleSigningAddress.ts:3]: Wrong constant name ORACLE_SA_PROD :)

Comment (daveroga) [reply scripts/setOracleSigningAddress.ts]: Yes, sorry

**Review by AndriianChestnykh — APPROVED** [review]

**Pull request merged**